src/applications/claims-status/components/ClaimLetterListItem.jsx

```diff
@@ -8,11 +8,21 @@ import environment from 'platform/utilities/environment';
 const downloadUrl = id => `${environment.API_URL}/v0/claim_letters/${id}`;
 
 const formatDate = date => {
-  return format(new Date(date), 'MMMM dd, yyyy');
+  // JavaScript dates have a funky behavior where passing in a string
+  // like this: '2022-11-18' will convert the string to a date in GMT
+  // and convert it to the users timezone. Passing in a string like
+  // this: '2022/11/18' however will be treated as midnight on 11/18/2022
+  // in the users timezone
+  const formattedDate = date.replace(/-/g, '/');
+
+  return format(new Date(formattedDate), 'MMMM dd, yyyy');
```

Comment:
I don't think we can rely on this, it may be a browser-specific behavior.

I recall needing to use `parse` and including the time to get it to properly format it, e.g.

```js
format(parse(`${date} 00:00:00`), 'MMMM dd, yyyy');
```

But I ended up going back to use moment JS for a forgotten reason